### ISSUE TITLE
Error : Type Arrow cannot be assigned to type Plugins

### ISSUE THREAD

**Issue by dukesx** [naver/egjs-flicking#558]
## Description
I am using Next JS 11.1 and i am trying to use Arrow plugin with Flicking Carousel (Current latest). It seems that as soon as i add new Arrow in an array to plugins property, flicking crashes and wont start. The error i get is 

```
Type 'Arrow' is not assignable to type 'Plugin'.
  Types of property 'init' are incompatible.
    Type '(flicking: import("/node_modules/@egjs/flicking/declaration/Flicking").default) => void' is not assignable to type '(flicking: import("/node_modules/@egjs/react-flicking/node_modules/@egjs/flicking/declaration/Flicking").default) => void'.
      Types of parameters 'flicking' and 'flicking' are incompatible.
        Type 'Flicking' is missing the following properties from type 'Flicking': _viewport, _camera, _control, _renderer, and 66 more.
```

Note 1: i am taking precautions mentioned for SSR as per documentation with hidebeforeinit etc
Note 2: There is no ViewportSlot exported as well from react flicking
issue on this line ----->    import Flicking, { ViewportSlot } from "@egjs/react-flicking";

** please update the example for Arrow plugin

## Steps to check or reproduce
1. use Nextjs 11.1
2. Add Flicking and its arrow plugin
3. Import Them and initialize like the example demo on this page: https://naver.github.io/egjs-flicking/Plugins#arrow
4. see error

**Comment by WoodNeck** [comment]
Hello @dukesx!
Looks like the versions are mismatching, can I ask which versions of Flicking-related packages you're using?

**Comment by dukesx** [comment]
> Hello @dukesx!
> Looks like the versions are mismatching, can I ask which versions of Flicking-related packages you're using?

You are right, I am using react flicking v3.7.3 from npm for 4.3.0 plugin version. It is strange that I just did npm install @egjs/react-flicking and it didn't install the latest, instead did the older version.

**Comment by WoodNeck** [comment]
Well there was one time that I published v3.7.3 as the latest version by mistake.
Maybe you had installed react-flicking that time, my mistake.

You can fix it by either updating the Flicking's version to the latest (recommended)
or downgrading the plugin's version to v3 :)

**Comment by dukesx** [comment]
Okay so i tried doing it and it broke the carousel. I have upgraded to 4.3.2. Also i get the same error about cannot assign type plugin to arrow.

**Comment by WoodNeck** [comment]
Well, the `gap` option is removed, which can be done by setting `margin-right` property on the panels on v4.
For the CSS I don't sure what's the proper placement of your panels on your site so I don't sure, but do you mean the panels should be aligned in a column, like using the `horizontal: false`?
